castle/cms/profiles/2_6_18/metadata.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<metadata>
+  <version>2618</version>
```

Comment:
I think the upgrade step looks good overall but we'll be on version 2.6.24 so we need to renumber/rename all the 2618 references.